### PR TITLE
[#151] Added ux-label component

### PR DESCRIPTION
**Pull request by zac1st1k**

### CODE REVIEW

File: src/components/ux-label/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+> ux-label Documentation
```
Comment (ScottMaclure) [inline src/components/ux-label/README.md]: Add notes as per ux-off-canvas:

http://ractive-foundation.github.io/ractive-foundation/ux-off-canvas.html

File: src/components/ux-label/ux-label.feature [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Feature: ux-label test suite
+
+  Scenario: Loading ux-label
+    Given I have loaded component "ux-label" with use case "UseCase"
```
Comment (ScottMaclure) [inline src/components/ux-label/ux-label.feature:4]: Add test for label's value.

Edit: Check with @nathf  first - he's working on the test suite - so you can probably skip this for now.

File: src/components/ux-label/ux-label.hbs [hunk 3]
```diff
@@ -0,0 +1 @@
+ <span class="{{class}} label">{{text}} Label</span>
```
Comment (ScottMaclure) [inline src/components/ux-label/ux-label.hbs]: Remove "Label"

File: src/components/ux-label/ux-label.feature (hunk 2 above)
Comment (zac1st1k) [inline src/components/ux-label/ux-label.feature:4]: Confirmed with @nathf, it can be skipped for now.

Comment (nathf) [inline src/components/ux-label/ux-label.feature:4]: Test suite is updated :+1:

**Pull request merged**